src/psyclone/domain/common/extract_driver_creator.py

```diff
@@ -643,25 +632,24 @@ def get_driver_as_string(self, nodes, input_list, output_list,
         :rtype: str
 
         '''
-        file_container = self.create(nodes, input_list, output_list,
+        file_container = self.create(nodes, read_write_info,
                                      prefix, postfix, region_name)
         return writer(file_container)
 
     # -------------------------------------------------------------------------
-    def write_driver(self, nodes, input_list, output_list,
-                     prefix, postfix, region_name, writer=FortranWriter()):
+    def write_driver(self, nodes, read_write_info, prefix, postfix,
+                     region_name, writer=FortranWriter()):
         # pylint: disable=too-many-arguments
         '''This function uses the `get_driver_as_string()` function to get a
         a stand-alone driver, and then writes this source code to a file. The
         file name is derived from the region name:
         "driver-"+module_name+"_"+region_name+".f90"
 
         :param nodes: a list of nodes.
-        :type nodes: list of :py:obj:`psyclone.psyir.nodes.Node`
-        :param input_list: list of variables that are input parameters.
-        :type input_list: list of :py:class:`psyclone.core.Signature`
-        :param output_list: list of variables that are output parameters.
-        :type output_list: list or :py:class:`psyclone.core.Signature`
+        :type nodes: List[:py:obj:`psyclone.psyir.nodes.Node`]
```

Comment:
Oops  - yes. I think I copy&pasted this from somewhere, and then it spread :)  I did notice quite a few other, unrelated files also having py:obj, but I didn't fix them (since in some cases py:obj is used like: `:py:obj:`node`).